style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #3c3a39;
+  height: auto;
+  justify-content: space-between;
+  flex: 1;
+  padding-bottom: 16px;
+}
```

Comment:
- [ ] Good job displaying the header section with flexbox 👍. However, the `height: auto;` you have here is not what is in the design on Figma and also the height auto makes the header not be like the Figma's. But after I added `height: 63px` and removed the `padding-bottom: 16px` I have a better result.

Your header design:
<img width="234" alt="Loom_XL6V07f64Q" src="https://user-images.githubusercontent.com/81468675/148192555-6154a537-fa90-462e-a97d-958018c0cabb.png">

After I fixed it:
<img width="235" alt="Loom_cxrqvfLki6" src="https://user-images.githubusercontent.com/81468675/148192622-88cde00b-e94e-485d-840f-a2a1646f7277.png">

Here is the changes I made to get that result:
```suggestion
.header {
  display: flex;
  align-items: center;
  background-color: #3c3a39;
  height: 63px;
  justify-content: space-between;
  flex: 1;
}
```